package.json

```diff
@@ -23,12 +23,14 @@
     "ts-node": "^10.9.1"
   },
   "devDependencies": {
+    "@cdklabs/cdk-enterprise-iac": "^0.0.303",
     "@types/jest": "^29.5.2",
     "@types/node": "*",
     "@typescript-eslint/eslint-plugin": "^5.50.0",
     "@typescript-eslint/parser": "^5.48.2",
     "aws-cdk": "^2.85.0",
     "aws-cdk-lib": "^2.85.0",
+    "cdk-ecr-deployment": "^2.5.30",
```

Comment:
Is this needed since `osml-cdk-constructs` already have this package?

Reply:
Yes the build fails without it